paddle/phi/core/dense_tensor.inl

```diff
@@ -127,7 +128,20 @@ following codes there.
 #ifdef PADDLE_WITH_MKLDNN
 
 public:
-inline dnnl::memory::format_tag format() const { return format_; }
+  dnnl::memory::desc mem_desc() const;// {
+  // for now some ops are using format and some are using mem_desc, so
+  // temporarily supporting both is needed
+//   return mem_desc_ ? mem_desc_
+//                    : dnnl::memory::desc(phi::vectorize(meta_.dims),
+//                                         paddle::framework::ToMKLDNNDataType(paddle::framework::TransToProtoVarType(meta_.dtype)), format_);
+// }
+
```

Comment:
will the commented lines be used later or deleted directly?

Reply:
I have deleted them